Unix/tools/auth_tests_setup.sh

```diff
@@ -37,34 +40,60 @@ do
 done
 
 #
-# if required env variables don't exist, then quit
+# Setup the NTLM auth file for client and server. They must be different files because 
+# if the differeing permissions, as the server file must be owned by root and the client by
```

Comment:
If we made the non-root changes, then we'd create an account (like "omiserver") on install, and files would be owned by that.

Give me a brief list of what to change if we do this work, and I'll put it in the spec. That will allow closure of this specific issue.